study-builder/studies/mbc/i18n/es.conf

```diff
@@ -1,1063 +1,698 @@
+"about-you_sections_01_blocks_09_nested_01_question_promptTemplate_01"="por favor liste los nombres de las terapias recibidas que hayan funcionado  excepcionalmente bien, si sabe los nombres. Para una lista de las terapias más comúnmente usadas, haga  clic aquí."
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 508
-"about-you_sections_01_blocks_09_nested_01_question_additionalItemTemplate_01"="[ES]Other Therapy"
+"about-you_sections_01_blocks_09_nested_01_question_additionalItemTemplate_01"="Otra terapia"
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 524
-"about-you_sections_01_blocks_09_nested_01_question_addButtonTemplate_01"="[ES]+ Add Another Therapy"
+"about-you_sections_01_blocks_09_nested_01_question_addButtonTemplate_01"="+ Agregar otra terapia"
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 544
-"about-you_sections_01_blocks_09_nested_01_question_children_01_promptTemplate_01"="[ES]Choose therapy..."
+"about-you_sections_01_blocks_09_nested_01_question_children_01_promptTemplate_01"="Elige la terapia..."
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 576
-"about-you_sections_01_blocks_10_question_promptTemplate_01"="[ES]If you have had an extraordinary response to therapy, tell us more about it."
+"about-you_sections_01_blocks_10_question_promptTemplate_01"="Si ha tenido una respuesta excepcional o duradera a alguna terapia, díganos más sobre esto."
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 591
-"about-you_sections_01_blocks_10_question_placeholderTemplate_01"="[ES]Describe response/therapy"
+"about-you_sections_01_blocks_10_question_placeholderTemplate_01"="Describa las Terapias"
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 615
-"about-you_sections_01_blocks_11_question_promptTemplate_01"="[ES]When was your most recent biopsy of your cancer?"
+"about-you_sections_01_blocks_11_question_promptTemplate_01"="¿Cuándo fue la biopsia más reciente de su cáncer de mama metastásico?"
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 640
-"about-you_sections_01_blocks_12_question_promptTemplate_01"="[ES]Tell us anything else you would like to about yourself or your cancer."
+"about-you_sections_01_blocks_12_question_promptTemplate_01"="Díganos en más detalle sobre usted o sobre su cáncer de mama metastásico:"
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 664
-"about-you_sections_01_blocks_13_question_promptTemplate_01"="[ES]In what year were you born?"
+"about-you_sections_01_blocks_13_question_promptTemplate_01"="¿En qué año nació?"
 # ddp-study-server/study-builder/studies/mbc/about-you.conf: 694
-"about-you_sections_01_blocks_15_question_promptTemplate_01"="[ES]What is your ZIP or postal code?"
+"about-you_sections_01_blocks_15_question_promptTemplate_01"="¿En qué país reside?"
```

Comment:
The Spanish text here asks: "In what country do you reside?" which does not match the request for ZIP or postal code,

Reply:
:man-facepalming: